tests/migration_test.go

```diff
@@ -932,23 +933,45 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		Context("with an Fedora shared NFS PVC, cloud init and service account", func() {
 			var pvName string
 			var vmi *v1.VirtualMachineInstance
-			BeforeEach(func() {
+
+			initNFS := func(isIpv6 bool) {
 				tests.SkipNFSTestIfRunnigOnKindInfra()
 				pvName = "test-nfs" + rand.String(48)
 				// Prepare a NFS backed PV
 				By("Starting an NFS POD")
 				os := string(cd.ContainerDiskFedora)
-				nfsIP := tests.CreateNFSTargetPOD(os)
+				nfsIPs := tests.CreateNFSTargetPOD(os)
 				// create a new PV and PVC (PVs can't be reused)
 				By("create a new NFS PV and PVC")
+				nfsIP := libnet.GetIp(libnet.GetPodIpsStrings(nfsIPs), isIpv6)
```

Comment:
why not to use primary and secondary in the tables ?
and then the primary takes IPs[0]
and secondary takes, if exists, IPs[1]
and if a family need some special treatment such as skip we simply add "if..."
then those functions arent needed, and its simpler imho.

Reply:
Because we want to be explicit we are checking ipv4 and ipv6. So that if somebody outside the team/kubevirt we look at the tests/test results he will be able to easily see which features are ipv4 supported and which are ipv6 supported,

Reply:
i think that as the future would support both, and that we can add By with the family under test, it would be still better to use primary and secodary

also since the code will check `if ip is v6 .... skip`
it would be explicit, just the explicitness would move from the table entry to the body of the test,
and will reduce the surround logic

Reply:
moreover having it in the table entry cant say if its skipped or not, so the user must look on the body, and then we will anyway have skip logic on both approaches.

please reconsider

Reply:
I think that would defeat the purpose we have of being clear that ipv4 / ipv6 ip families are being tested.

Also, adding yet another abstract concept that we would need to explain to everybody seems counter-productive.

Reply:
you can still print the family
lets say you will have singleton func in a lib that you call it in the table entry (assuming sprintf is allowed there),
Family(PRIMARY) or Family(SECONDARY)
and it would return you the string of the family, based on what it checked in a virt-handler

Reply:
We discussed the approach of changing the description of the test. Currently it is not supported.
We need the flsgs to be initialized when we try to check the ips of virt handler (the get the path of kubeconfig). And in our current ginko version, when the tests tree is built, the flags are not initialized yet.
Anyway, currently we don't have primary and secondary ips. The ipv4 is always the primary. So I don't think that it is relevant for now.

* Regarding the `By`- afaik, it is not printed in a successful test summary (it is not part of the test tree).

Reply:
> We discussed the approach of changing the description of the test. Currently it is not supported.
> We need the flsgs to be initialized when we try to check the ips of virt handler (the get the path of kubeconfig). And in our current ginko version, when the tests tree is built, the flags are not initialized yet.

regarding this, maybe os.Getenv("KUBECONFIG") would be enough for the other flags discussion if any.

> Anyway, currently we don't have primary and secondary ips. The ipv4 is always the primary. So I don't think that it is relevant for now.
> 
> * Regarding the `By`- afaik, it is not printed in a successful test summary (it is not part of the test tree).